src/mail /mail.module.ts

```diff
@@ -0,0 +1,14 @@
+import { MailerModule } from '@nestjs-modules/mailer';
+import { HandlebarsAdapter } from '@nestjs-modules/mailer/dist/adapters/handlebars.adapter';
+import { Module } from '@nestjs/common';
+import { join } from 'path';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { mailerAsyncConfig } from 'src/configs/mailer/mailer-config';
+import { MailService } from './mail.service';
+
+@Module({
+  imports: [MailerModule.forRootAsync(mailerAsyncConfig)],
```

Comment:
I think there are no `async` code in configs

```suggestion
  imports: [MailerModule.forRoot(mailerConfig)],
```